### PR TITLE
The links when hovered give better UI

### PR DESCRIPTION
**Pull request by aashimgarg**
The links when hovered give better ui as now i have given proper padding at various places in the html webpage.
issue solved #98

### CODE REVIEW

**Comment by aashimgarg** [comment]
@jobara please review it?

**Comment by jobara** [comment]
@aashimgarg I'm not sure what issue this is actually addressing. I've left a comment on the related issue asking for more details.

File: css/style.css [hunk 1]
```diff
@@ -541,6 +543,10 @@ figcaption details p {
     background-color: #000000;
 }
 
+.floe-footer .floe-content a{
+    padding : 0px 3.5px ;
+}
+
```
Comment (jobara) [inline css/style.css]: Please leave a space between selectors and the brace "{" for example `a {`.

Also seems we are using em values for padding not px. Please correct throughout.

**Comment by aashimgarg** [comment]
Fixed changes @jobara !

File: index.html [hunk 2]
```diff
@@ -84,9 +84,9 @@ <h1><a rel="home" title="Home" href="index.html" class="floe-header-logo">floe
                 <div class="columns small-12 medium-12 large-5 align-self-bottom">
                     <nav class="floe-nav">
                         <ul>
-                            <li><a href="./news/index.html">News</a></li>
-                            <li><a href="./resources.html">Resources</a></li>
-                            <li><a href="./ui-options.html">UI Options</a></li>
+                            <li><a href="./news/index.html">News&nbsp;</a></li>
+                            <li><a href="./resources.html">Resources&nbsp;</a></li>
+                            <li><a href="./ui-options.html">UI Options&nbsp;</a></li>
```
Comment (jobara) [inline index.html]: You should not be adding `&nbsp;` into the markup. Styling should be done with CSS.

**Comment by jobara** [comment]
@aashimgarg Thanks for the contribution; however, I don't think the issue has been fully discussed yet.

**Comment by sachin10101998** [comment]
It can be closed without merging.

**Comment by greatislander** [comment]
This PR won't be merged as we are currently beginning a redesign process. Thanks for your contribution!

**Pull request closed without merging**